client-app/src/examples/portfolio/detail/OrdersPanelModel.js

```diff
@@ -90,8 +114,12 @@ export class OrdersPanelModel extends HoistModel {
         }
 
         try {
-            const orders = await XH.portfolioService.getOrdersAsync({positionId, loadSpec});
-            gridModel.loadData(orders);
+            const orders = await XH.portfolioService.getOrdersAsync({positionId, loadSpec}),
+                symbols = uniq(map(orders, 'symbol')),
+                sparklineSeries = await XH.portfolioService.getSparklineSeriesAsync({symbols, loadSpec}),
+                rows = orders.map(order => ({...order, tradingVolume: sparklineSeries[order.symbol]}));
```

Comment:
nit:  typically would just mutate the order, by installing the tradingVolume rather than spreading....